tests/schema/test_generate_schema.py

```diff
@@ -186,13 +186,13 @@ async def test_schema(self):
 CREATE TABLE "tournament" (
     "tid" INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
     "name" VARCHAR(100) NOT NULL  /* Tournament name */,
-    "created" TIMESTAMP NOT NULL  /* Created *\\/'`\\/* datetime */
+    "created" TIMESTAMP NOT NULL  /* Created *\\/'`\\/* datetime */ DEFAULT CURRENT_TIMESTAMP
```

Comment:
Could we move the default to before the comment? It currently reads rather badly?

Reply:
The ``FIELD_TEMPLATE = '"{name}" {type} {nullable} {unique}{primary}{comment}{default}``, so...

Reply:
You can override it on a per-dialect level. add a `FIELD_TEMPLATE` in `SqliteSchemaGenerator` and it will use that instead.

Reply:
Also, the comment is always a blank string for Postgresql as it emits it in a later instruction. And it is overridden for MySQL. So you can happily change the default?

Reply:
Hold on, did you not add that in your previous pr?

Reply:
What? You mean move all dialect  ``default`` previous ``comment``?

Reply:
I meant you added it in your previous PR about this:
https://github.com/tortoise/tortoise-orm/commit/72f84f0848dc68041157f03e60cd1c92b0ee5137#diff-738f1a7c384904401ec302972c0c4c21R21

So you can change the one in `base/schema_generator.py` and it will fix SQLite, and have no change for PostgreSQL (since PostgreSQL does comments outside of the `create table` statement)